src/modules/battery.cpp

```diff
@@ -1,9 +1,15 @@
 #include "modules/battery.hpp"
-
+#if defined(__FreeBSD__)
+// clang-format off
```

Comment:
`clang-format` lines can be dropped now as there's only one header, so no need to interfere with sorting by name.